docs/books/nvchad/vale_nvchad.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Using vale in NvChad
+author: Steven Spencer
+contributors: Franco Colussi
+tags:
+  - vale
+  - linters
+---
+
+# `vale` in NvChad (Neovim)
+
+## Introduction
+
+`vale.sh` is one of the foremost open source projects for technical writers looking to improve their voice and style consistency. It can be used with a number of editors on nearly every major OS platform (Linux, MacOS, Windows). You can find out more about the project by heading up to [the vale.sh website](https://vale.sh/). This guide is going to walk you through adding `vale` to NvChad. Since it is included in the Mason packages for install, the process is not to difficult, although it does involve some minor editing and configuration to get things going. To be clear, NvChad is really the configuration manager for the editor Neovim, so from this point forward, we will be referencing `nvim`.
```

Comment:
```suggestion
`vale.sh` is one of the foremost open source projects for technical writers looking to improve their voice and style consistency. It can be used with a number of editors on nearly every major OS platform (Linux, MacOS, Windows). You can find out more about the project by heading up to [the vale.sh website](https://vale.sh/). This guide is going to walk you through adding `vale` to NvChad. Since it is included in the Mason packages for install, the process is not too difficult, although it does involve some minor editing and configuration to get things going. To be clear, NvChad is really the configuration manager for the editor Neovim, so from this point forward, we will be referencing `nvim`.
```